### PR TITLE
WIP: Allow user to pass options to pymongo find.

### PR DESCRIPTION
**Pull request by danielballan**
One option of interest is `no_cursor_timeout`. I believe the cursor timeouts
were encountered at XPD or PDF (by @tacaswell? or @mrakitin?). I just
encountered them while trying to do a long export task at FXI.

I think we ought to leave timeouts on by default---they are there for a reason---
but we clearly need the ability to request a timeout-free cursor for the
server for some cases.

This should be documented and tested before being merged.

### CODE REVIEW

File: databroker/_drivers/mongo_normalized.py [hunk 1]
```diff
@@ -204,6 +209,7 @@ def __init__(self, metadatastore_db, asset_registry_db, *,
         self._metadatastore_db = mds_db
         self._asset_registry_db = assets_db
         self._query = query or {}
+        lse.f_find_kwargs = find_kwargs or {}
```
Comment (mrakitin) [inline databroker/_drivers/mongo_normalized.py]:
```suggestion
        self._find_kwargs = find_kwargs or {}
```

File: databroker/_drivers/mongo_normalized.py [hunk 2]
```diff
@@ -300,6 +308,7 @@ def search(self, query):
             metadatastore_db=self._metadatastore_db,
             asset_registry_db=self._asset_registry_db,
             query=query,
+            find_kwargs=find_kwargs,
```
Comment (mrakitin) [inline databroker/_drivers/mongo_normalized.py]: Should it be `kwargs`?

Comment (danielballan) [reply databroker/_drivers/mongo_normalized.py]: Yes, good catch.

**Comment by danielballan** [comment]
Fixed and force-pushed. Also added a test.

**Pull request merged**